client/components/checklist/checklist.js

```diff
@@ -38,7 +38,9 @@ export default class Checklist extends PureComponent {
 
 	calculateCompletion() {
 		const { children } = this.props;
-		const childrenArray = Children.toArray( children );
+		const childrenArray = Children.toArray( children ).filter(
+			task => task && task.props && ! task.props.noCount
```

Comment:
I wonder if there could be a more descriptive name for this prop than `noCount`? Not a blocker at all since it might be temporary anyway.

`excludedFromTaskCount` ...lengthy?

Reply:
Good point. I'm hoping this is short lived, it's not a good long-term solution for a shared component as I tried to note in the README change, but the name could be more descriptive.

I'll change it to `excludeFromCount`.

Reply:
c1b5c244ccd87fb227e3376c74c17ecb3d71646b